src/CLIFramework/IO/StreamWriter.php

```diff
@@ -11,6 +11,10 @@ class StreamWriter implements Writer
 
     public function __construct($stream)
     {
+        if (!is_resource($stream)) {
+            throw new \RuntimeException("invalid stream");
+        }
```

Comment:
There is no need to add `use RuntimeException` because the reference of `RuntimeException` is global reference like `\RuntimeException`. But do you prefer to add the `use` statement on the top?